### PR TITLE
Remove Blockly bindings for App Lab and Game Lab

### PR DESCRIPTION
**Pull request by joshlory**
We only use App Lab and Game Lab with Droplet, but some old bindings for Blockly still exist.  It's not clear if these bindings are up-to-date since they're not used anywhere or tested.

This PR removes the Blockly bindings for App / Game Lab and ensures they're used with Droplet.

### CODE REVIEW

File: apps/test/integration/util/runLevelTest.js [hunk 1]
```diff
@@ -92,22 +92,6 @@ StubDialog.prototype.show = function () {
...
-  require('@cdo/apps/maze/main');
-  require('@cdo/apps/flappy/main');
-  require('@cdo/apps/turtle/main');
-  require('@cdo/apps/eval/main');
-  require('@cdo/apps/studio/main');
-  require('@cdo/apps/calc/main');
-  require('@cdo/apps/bounce/main');
-  require('@cdo/apps/applab/main');
-  require('@cdo/apps/gamelab/main');
-  require('@cdo/apps/craft/main');
```
Comment (islemaster) [inline apps/test/integration/util/runLevelTest.js:108]: Is this related to removing blockly bindings, or just doing other cleanup at the same time?  How did you verify that this isn't needed under webpack?

Comment (joshlory) [reply apps/test/integration/util/runLevelTest.js:108]: Unrelated cleanup.  Verified by checking that `grunt test` still passes.  Is there anything else I should check?

Comment (islemaster) [reply apps/test/integration/util/runLevelTest.js:108]: No, that should be sufficient.

**Comment by islemaster** [comment]
I love it.  I'm pulling the branch to do some manual testing of my own - I'm a little nervous because I think there are places where we refer to droplet blocks simply as 'blocks' and want to be certain you haven't removed any of those.  I'll sign off when I'm done.

**Comment by islemaster** [comment]
LGTM!

image

You get the bubble award for minimizing our surface area today.

**Pull request merged**